R/diagnostics.R

```diff
@@ -19,6 +19,10 @@
 #'   tolerance set by {tol}, FALSE otherwise.
 #' @export
 isTimestepRegular <- function(dates, hist=TRUE, tol=.Machine$double.eps^0.5, handler=stop) {
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes=TimeInterval=geom_histogram=xlab=NULL
```

Comment:
i agree that TimeInterval is probably the second type of issue. my best guess without directly experimenting is that it's because of `aes(x=TimeInterval)`, where it's not obvious to R CMD check that `TimeInterval` is a column name within a data.frame. and, yes, let's set it it a value just before the ggplot call. But let's do
```r
TimeInterval  <- '.ggplot.var'
```
instead of NULL, please.